manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml

```diff
@@ -9,62 +9,70 @@ spec:
+    The Jupyter spawner supports your Data Scientist explorations and model deployments.
 
-    We just know you will absolutely love this amazing environment.  This quick start will get you working in a notebook in just a few minutes.
+    We just know you will absolutely love this amazing environment.  This quick start gets you working in a notebook in just a few minutes.
   tasks:
-    - title: Launch JupyterHub
+    - title: Create a Data Science project
       description: |-
-        ### To find the JupyterHub Launch action:
-        1. Click **Applications** &#x2192; **Enabled**.
-        2. Find the JupyterHub card.
-        3. Click **Launch** on the JupyterHub card to access the JupyterHub **Start a notebook server** page.
-
-        A new browser tab will open displaying the **Start a notebook server** page.
+        ### Create a project
+        Creating a project helps you organize your work in one place.
+        1. Click **Data Science Projects**.
+        2. Click **Create a Data Science project**.
+        3. Type a name for your project.
+        4. Click **Create**.
       review:
         instructions: |-
-          #### To verify you have launched JupyterHub:
-          Is a new **JupyterHub** browser tab visible with the **Start a notebook server** page open?
+          #### To verify you have created a project:
+          Does your new project appears in the list of Data Science projects?
         failedTaskHelp: This task is not verified yet. Try the task again.
       summary:
-        success: You have launched JupyterHub.
+        success: You have created a new project.
         failed: Try the steps again.
-    - title: Configure and start an environment
+    - title: Create a workbench
       description: |-
-        ### Configuring and starting an environment:
-        1. Select a notebook image from the options provided.
-        2. Select a container size based on your computation needs.
-        3. Click the **Start** button.
-
-        The **Start a notebook server** page will reload and indicate that the system is starting up.
+        ### Create a workbench
+        A workbench provides an isolated area for you to examine and work with data models. When you configure a workbench, you create a new Jupyter notebook from an existing notebook container image to access its resources and properties. 
+        1. In the list of **Data Science Projects**, click on your project.
+        2. Click **Create workbench**.
+        3. Configure the properties of the workbench you are creating.
+           a. Type a name and a description for your workbench.
+           b. Select a notebook image to use for your workbench server.
+           c. Select the container size for your server.
+           d. To enable data science pipelines in JupyterLab, create a **Config Map** environment variable with the following Key/Value pair: 
```

Comment:
@Jooho @HumairAK is it still necessary to set env PIPELINES_SSL_CA_CERTS or does odh notebook controller do that?

Reply:
It is still needed if the cluster is an insecure cluster
as the workaround in the standard notebook workflow
The same is being used here.

Reply:
I see, thank you.